Ginger/Ginger/Run/RunSetsExecutionsHistoryPage.xaml.cs

```diff
@@ -251,13 +252,15 @@ private void DeleteExecutionReports(System.Collections.IList runSetReports)
                     LiteDbConnector dbConnector = new LiteDbConnector(Path.Combine(mRunSetExecsRootFolder, "GingerExecutionResults.db"));
                     dbConnector.DeleteDocumentByLiteDbRunSet(filterData[0]);
                 }
-                else
+                else if (remoteDeletionFlag == false)
                 {
-                    string runSetFolder = executionLoggerHelper.GetLoggerDirectory(runSetReport.LogFolder);
+                    Reporter.ToUser(eUserMsgKey.RemoteExecutionResultsCannotBeAccessed);
+                    remoteDeletionFlag = true;
```

Comment:
please remove the comments as it was being used for TextFile type method as well. Add specific condition for Remote.